packages/graphql-server/src/functions/graphql.ts

```diff
@@ -373,3 +374,15 @@ export const createGraphQLHandler = ({
     }
   }
 }
+
+export const generateGraphiQLHeader = () => {
+  let header
+  if (process.env.NODE_ENV === 'development') {
+    try {
+      header = require('api/dist/lib/generateGraphiQLHeader').default()
+    } catch (err) {
+      console.log('Could not find generateGraphiQLHeader')
+    }
+  }
+  return header
+}
```

Comment:
@dac09  I added the dev env check in the graphql-server package. This works in my dev testing environment but I need to merge to test in CI. You mentioned there might be issues with pods or something when trying to access `require('api/dist/lib/generateGraphiQLHeader')` from the library? Can you give me a bit more context? 
Do you think this will not work in prod even though it works in dev?